proptest/src/arbitrary/_std/path.rs

```diff
@@ -11,13 +11,121 @@
 
 use std::path::*;
 
-// TODO: Figure out PathBuf and then Box/Rc/Box<Path>.
+use crate::{
+    arbitrary::{SMapped, StrategyFor},
+    path::PathParams,
+    prelude::{any, any_with, Arbitrary, Strategy},
+    std_facade::{string::ToString, Arc, Box, Rc, String, Vec},
+    strategy::{statics::static_map, MapInto},
+};
 
 arbitrary!(StripPrefixError; Path::new("").strip_prefix("a").unwrap_err());
 
+/// A private type (not actually pub) representing the output of [`PathParams`] that can't be
```

Comment:
non-blocking: it is actually pub(?) not sure if this just means it isn't intended to be publicly used or if this is residual from previous iterations.

Reply:
I meant that even though it is pub, it is not exported as an API so that backwards compatibility is maintained. Any suggestions for better phrasing?

Reply:
I don't think its a huge deal either way. I would probably just say something like:
`"An opaque type representing the output of ...`